jakarta.ls/src/main/java/org/eclipse/lsp4jakarta/api/JakartaLanguageClientAPI.java

```diff
@@ -49,6 +50,16 @@ default CompletableFuture<List<PublishDiagnosticsParams>> getJavaDiagnostics(Jak
     default CompletableFuture<List<String>> getContextBasedFilter(JakartaClasspathParams classpathParams) {
         return CompletableFuture.completedFuture(null);
     }
+    
```

Comment:
I don't think microprofile/java/javaCursorContext and jakarta/java/completion messages are required so we can remove these two entries.